### PR TITLE
[Term Entry] Kotlin `hypot()` method

### LINKED ISSUE #3525: [Term Entry] Kotlin Math .hypot()
### Reason/inspiration (optional)

We would like a new term entry on the `.hypot()` method for Kotlin: [Math](https://www.codecademy.com/resources/docs/kotlin/math-methods). The entry should go in a new file under `docs/content/kotlin/concepts/math-methods/terms/hypot/hypot.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```kotlin `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by manviii27**
### Description

A new term entry for Kotlin `.hypot()` method.
Close #3525 

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/math-methods/terms/hypot/hypot.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+The **`.hypot()`** method in Kotlin's `math` class is used to calculate the hypotenuse of two numbers in a way that avoids overflow and underflow. It returns the square root of the sum of the squares of its two arguments. This is also known as the Euclidean norm of a two-dimensional vector.
+
+## Syntax
+
+```pseudo
+hypot(x,y)
+```
+
+- x: One of the two numbers whose hypotenuse is to be calculated.
+- y: Another number whose hypotenuse is to be calculated.
```
Comment (SSwiniarski) [inline content/kotlin/concepts/math-methods/terms/hypot/hypot.md:24]:
```suggestion
- `y`: The other number whose hypotenuse is to be calculated.

```

File: content/kotlin/concepts/math-methods/terms/hypot/hypot.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+The **`.hypot()`** method in Kotlin's `math` class is used to calculate the hypotenuse of two numbers in a way that avoids overflow and underflow. It returns the square root of the sum of the squares of its two arguments. This is also known as the Euclidean norm of a two-dimensional vector.
+
+## Syntax
+
+```pseudo
+hypot(x,y)
+```
+
+- x: One of the two numbers whose hypotenuse is to be calculated.
```
Comment (SSwiniarski) [inline content/kotlin/concepts/math-methods/terms/hypot/hypot.md:23]:
```suggestion
- `x`: One of the two numbers whose hypotenuse is to be calculated.

```

File: content/kotlin/concepts/math-methods/terms/hypot/hypot.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+}
+```
+
+The output of this code will be:
+
+```shell
+5.0
+```
+
+## Codebyte Example
```
Comment (SSwiniarski) [inline content/kotlin/concepts/math-methods/terms/hypot/hypot.md:49]: This either needs to be changed to a regular example or removed, the codebytes engine does not support Kotlin at this time.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@manviii27 I did a first review and left comments. The most important is you need to change the codebyte section because codebytes does not support Kotlin right now.

**Comment by manviii27** [comment]
@SSwiniarski I've made the changes, please review!

**Review by SSwiniarski — APPROVED** [review]
@manviii27 looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@manviii27 @SSwiniarski Just made a couple of direct edits to update some formatting . Looks good to merge

**Pull request merged**